### DISCUSSION TITLE
Image Rom Scrapper

### DISCUSSION THREAD

**Discussion by pirex360** [OnionUI/Onion#524] [Ideas]
Hi, 

This product is fantastic, but without WIFI it's difficult to prepare the roms folder with the images and metadata.
One great solution for this, if someone is up to it, is creating software that we point to a disk location or sd location, and it automatically parses the files and puts the images in the right folders and the right names.

I think this could be done.... is there anything like this out there?

**Comment by Xaumir** [comment]
There is a windows/linux app called Skraper that does exactly that, take a look http://skraper.net/

**Comment by Olywa** [comment]
There is also a nice guide for how to use Skraper already in the Onion Wiki here:
https://github.com/jimgraygit/Onion/wiki/3.-FAQ-(Frequently-asked-questions)

**Reply by tooonyo** [reply]
I confirm it's a wonderful software but I noticed that the images don't appear for all my games in folders (I createds folders to sort them by genre). Scraper create the same structure that my game (the folders) in the Images folder but it still doesn't work. Do you think it's a Onion functionality or Miyoo OS ? Is it working on you console ? Thanks

**Reply by Olywa** [reply]
You can only have one level deep for subfolders (a limitation of the Miyoo firmware, not OnionOS) but images can be used. They must be in the systems root Imgs folder, so it should look like:

->Roms
---->MD
-------->Imgs   (put images for ALL megadrive/genesis roms here)
-------->Beat em up
-------------->rom 1
-------------->rom 2
-------->Shoot em up
-------------->rom 1
-------------->rom 2

NOTE: You cannot use gamelists with subfolders as they will just be presented as one single list 
(another limitation of the Miyoo firmware fw).
